From d103e04ba7f3c06640c3dd0e92d6076506d429a2 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Fri, 14 Sep 2018 16:44:53 +0200
Subject: [PATCH 0398/1852] media: platform: xilinx: Fix Kconfig indentation

commit 4cda7fa23363a25616f1c58427133e8c54a0758d from
https://github.com/Xilinx/linux-xlnx.git

Use proper indentation for Kconfig fragments - help part.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/Kconfig | 64 +++++++++++++--------------
 1 file changed, 32 insertions(+), 32 deletions(-)

diff --git a/drivers/media/platform/xilinx/Kconfig b/drivers/media/platform/xilinx/Kconfig
index e972856dea35..85f37244e210 100644
--- a/drivers/media/platform/xilinx/Kconfig
+++ b/drivers/media/platform/xilinx/Kconfig
@@ -16,114 +16,114 @@ config VIDEO_XILINX_AXI4S_SWITCH
 	tristate "Xilinx AXI4-Stream Video Switch"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx AXI4-Stream Video Switch. This is a
-	   V4L sub device based driver. It supports fixed (TDEST based)
-	   as well as dynamic (control register based) routing.
-	   Say M to modularize. Say N if unsure.
+	  Driver for the Xilinx AXI4-Stream Video Switch. This is a
+	  V4L sub device based driver. It supports fixed (TDEST based)
+	  as well as dynamic (control register based) routing.
+	  Say M to modularize. Say N if unsure.
 
 config VIDEO_XILINX_CFA
 	tristate "Xilinx Video Color Filter Array"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Color Filter Array
+	  Driver for the Xilinx Color Filter Array
 
 config VIDEO_XILINX_CRESAMPLE
 	tristate "Xilinx Video Chroma Resampler"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Chroma Resampler
+	  Driver for the Xilinx Chroma Resampler
 
 config VIDEO_XILINX_DEMOSAIC
 	tristate "Xilinx Video Demosaic IP"
 	depends on VIDEO_XILINX
 	---help---
-	Driver for Xilinx Video Demosaic IP. This is a V4L sub-device
-	based driver for the Demosaic IP that takes input a Bayer video
-	stream format as input and generates an RGB video output.
-	Say M to modularize. Say N if unsure.
+	  Driver for Xilinx Video Demosaic IP. This is a V4L sub-device
+	  based driver for the Demosaic IP that takes input a Bayer video
+	  stream format as input and generates an RGB video output.
+	  Say M to modularize. Say N if unsure.
 
 config VIDEO_XILINX_GAMMA
 	tristate "Xilinx Gamma Correction LUT"
 	depends on VIDEO_XILINX
 	---help---
-	Driver for Xilinx Gamma Correction LUT IP. This is a V4L sub-device
-	based driver that exposes V4L controls to adjust Red, Blue and Green
-	Gamma Correction.
+	  Driver for Xilinx Gamma Correction LUT IP. This is a V4L sub-device
+	  based driver that exposes V4L controls to adjust Red, Blue and Green
+	  Gamma Correction.
 
-	Say M to modularize. Say N if unsure.
+	  Say M to modularize. Say N if unsure.
 
 config VIDEO_XILINX_HLS
 	tristate "Xilinx Video HLS Core"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Video HLS Cores
+	  Driver for the Xilinx Video HLS Cores
 
 config VIDEO_XILINX_REMAPPER
 	tristate "Xilinx Video Remapper"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Video Remapper
+	  Driver for the Xilinx Video Remapper
 
 config VIDEO_XILINX_RGB2YUV
 	tristate "Xilinx Video RGB to YUV Convertor"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Video RGB to YUV Convertor
+	  Driver for the Xilinx Video RGB to YUV Convertor
 
 config VIDEO_XILINX_SCALER
 	tristate "Xilinx Video Scaler"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Video Scaler
+	  Driver for the Xilinx Video Scaler
 
 config VIDEO_XILINX_SDIRXSS
 	tristate "Xilinx SDI Rx Subsystem"
 	depends on VIDEO_XILINX
 	---help---
-	Driver for Xilinx SDI Rx Subsystem
+	  Driver for Xilinx SDI Rx Subsystem
 
 config VIDEO_XILINX_SWITCH
 	tristate "Xilinx Video Switch"
 	depends on VIDEO_XILINX
 	---help---
-	   Driver for the Xilinx Video Switch
+	  Driver for the Xilinx Video Switch
 
 config VIDEO_XILINX_TPG
 	tristate "Xilinx Video Test Pattern Generator"
 	depends on VIDEO_XILINX
 	select VIDEO_XILINX_VTC
 	help
-	   Driver for the Xilinx Video Test Pattern Generator
+	  Driver for the Xilinx Video Test Pattern Generator
 
 config VIDEO_XILINX_VPSS_CSC
 	tristate "Xilinx VPSS CSC"
 	depends on VIDEO_XILINX
 	---help---
-	Driver for the Xilinx Video Processing Sub-System (VPSS)
-	Color Space Conversion. The driver provides RGB to YUV444
-	conversion and provides video controls like Brightness,
-	Contrast, Color Gains that can be applied to video.
-	Say N if unsure. Say M to modularize.
+	  Driver for the Xilinx Video Processing Sub-System (VPSS)
+	  Color Space Conversion. The driver provides RGB to YUV444
+	  conversion and provides video controls like Brightness,
+	  Contrast, Color Gains that can be applied to video.
+	  Say N if unsure. Say M to modularize.
 
 config VIDEO_XILINX_VPSS_SCALER
 	tristate "Xilinx Video VPSS Scaler"
 	depends on VIDEO_XILINX
 	---help---
-	Driver for Xilinx Video Processing Sub-System(VPSS) Scaler.
-	It allows upscaling and downscaling of video. It also supports
-	limited Color Space Conversion.
-	Say N if unsure.
+	  Driver for Xilinx Video Processing Sub-System(VPSS) Scaler.
+	  It allows upscaling and downscaling of video. It also supports
+	  limited Color Space Conversion.
+	  Say N if unsure.
 
 config VIDEO_XILINX_VTC
 	tristate "Xilinx Video Timing Controller"
 	depends on VIDEO_XILINX
 	help
-	   Driver for the Xilinx Video Timing Controller
+	  Driver for the Xilinx Video Timing Controller
 
 config VIDEO_XILINX_CSI2RXSS
 	tristate "Xilinx CSI2 Rx Subsystem"
 	depends on VIDEO_XILINX
 	---help---
-	Driver for Xilinx MIPI CSI2 Rx Subsystem
+	  Driver for Xilinx MIPI CSI2 Rx Subsystem
 
 endif #VIDEO_XILINX
-- 
2.31.1

